modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,299 @@
+metadata:
+  name: config
+spec:
+  profile: all
+  targetNamespace: openshift-pipelines
+  pipeline:
+    params:
+    - name: enableMetrics
+      value: "true"
+...
+----
+
+* Eventlistener OpenCensus metrics, which captures metrics at process level, has been added.
+
+* Triggers now has label selector; you can configure triggers for an event listener using labels.
+
+* The `ClusterInterceptor` custom resource definition for registering interceptors is added, which allows you to register new `Interceptor` types that you can plug in. In addition, the following relevant changes are made:
+
+** In the trigger specifications, you can configure interceptors using a new API that includes a `ref` field to refer to a cluster interceptor. In addition, you can use the `params` field to add parameters that will be passed on to the interceptors for processing.
+
+** The bundled interceptors CEL, GitHub, GitLab, and BitBucket, have been migrated. They are implemented using the new `ClusterInterceptor` custom resource definition.
+
+** Core interceptors are migrated to the new format, and any new triggers created using the old syntax will automatically switch to the new `ref` or `params` based syntax.
+
+* To disable prefixing the name of the task or step while displaying logs, you can use the `--prefix` option for `log` commands.
+
+* To display the version of a specific component, use the new `--component` flag in the `tkn version` command.
+
+* The `tkn hub check-upgrade` command is added, and other commands are revised to be based on the pipeline version. In addition, catalog names are displayed in the `search` command output.
+
+* Support for optional workspaces are added to the `start` command.
+
+* The plugins will be looked up in the current path, if they are not present in the `plugins` directory.
+
+* The `tkn [task | clustertask | pipeline] start` command will start interactively and ask for the `params` value, if default is not specified or passed through flag.
+
+* The `version` field is added in the `tkn task describe` command.
+
+* The option to automatically select `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener`, has been added in the `describe` command, if only one is present.
+
+* In the `tkn pr describe` command, a section for skipped tasks is added.
+
+* Support for the `tkn clustertask logs` has been added.
+
+* The YAML merge and variable from `config.yaml` has been removed. In addition, the `release.yaml` file can now be more easily consumed by tools like `kustomize` and `ytt`.
+
+* The support for resource names to contain the dot character (".") has been added.
+
+* The `hostAliases` in the `PodTemplate` specification has been added to the pod-level override of hostname resolution. It ha been achieved by modifying the `/etc/hosts`.
+
+* Steps in a task no longer receive a default `HOME` environment variable of `/tekton/home`, nor a default working directory of `/workspace`, as it is disabled.
```

Comment:
This is from 1.6, for 1.5 we will still be having old behaviour
cc: @vdemeester @nikhil-thomas

Reply:
True, this line is not needed in 1.5.